### DISCUSSION TITLE
POC implemented

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-AutoUpdate#72] [Show and tell]
Thought I should share (notice the WAU icon) - I should fill in the link in SCCM too 👍 :

![POC](https://user-images.githubusercontent.com/102996177/166713176-1dc09ab4-4b7d-49ee-b408-83a8e0dc387b.png)
![WAU](https://user-images.githubusercontent.com/102996177/166713724-181099f0-9317-429c-ab6d-eed631a9e233.png)

**Comment by KnifMelti** [comment]
Done:

![SCClient_Z9HjuUGncB](https://user-images.githubusercontent.com/102996177/166805660-a63aa90c-0f26-4fa0-bc41-864b619d6f30.png)
![SCClient_UqtqYIFXd7](https://user-images.githubusercontent.com/102996177/166805720-66c92582-2af2-4955-aef5-98204d0fef31.png)

**Reply by Romanitho** [reply]
haha. looks nice :p
Hope it works as expected :)

**Reply by KnifMelti** [reply]
Yup, **_it just works_**.. ..after I rewrote the [detection](https://github.com/Romanitho/Winget-Install/issues/33)... :)

**Comment by KnifMelti** [comment]
When this **POC** of ours is verified working as it should and the new **winget** is released as stable we're going to implement this in our whole environment (**Användare=Users**) except servers with **Winget-AutoUpdate** own update checking enabled but **NotificationLevel=None** - so don't you now feel any pressure ;):

![image](https://user-images.githubusercontent.com/102996177/167250078-0eb6f164-a65a-4736-af9b-e056dda5444d.png)

**Reply by Romanitho** [reply]
I won't push any new version without any triple checking 😅
I have implemented it to some of my customers as well ;)

**Comment by KnifMelti** [comment]
As from today.. ..I've implemented [Winget-Install v1.8.1](https://github.com/Romanitho/Winget-Install/releases/tag/v1.8.1) in our converting task from traditional SCCM applications to Winget applications in SCCM/Intune.
Out of the **570+** applications in SCCM I've identified **105** that can be **whitelisted** and replaced by Winget installs!

The conversion has started...
...and the next week we'll have an internal meeting deciding the installation parameters etc...

By then, [Winget-AutoUpdate (WAU) v1.15.0+](https://github.com/Romanitho/Winget-AutoUpdate/releases/tag/v1.15.0) is probably released as a stable/latest version and I'll be able to roll out this version as a preparation to our **5000+** clients!

Waiting for Microsoft to implement the **native Winget deployment** in **Intune** for **Microsoft Store** (Business/Education)... 💖

No pressure @Romanitho... 😜

**Reply by Romanitho** [reply]
Haha, I'll send you the bill 😜 
No pressure at all btw, its an open source project, provided "as is" 😅
But it was born for that, so I'm happy it is used even in a quite big environment that prove it works well 😇
And finally, you actively help in this project by testing and implementing features so, thank you 😉

**Reply by Romanitho** [reply]
And for WAU 1.15.0 pre-release, I think i will fix one or two stuffs but we can release it soon
